File: translation_output/misleading_test/wizardcoder/codenet/java/python-sanitized/p04030.py
import java.io.IOException;
import java.io.InputStream;
import java.util.NoSuchElementException;

class p04030:
    def main(self):
        sc = FastScanner()
        s = sc.next()
        t = ""
        for i in range(s.length()):
            if s.charAt(i)!= 'B':
                t += s[i]
            elif t == "":
                continue
            else:
                t = t[:-1]
        print(t)

class FastScanner:
    def __init__(self):
        self.in = System.in
        self.buffer = bytearray(1024)
        self.ptr = 0
        self.buflen = 0

    def hasNextByte(self):
        if self.ptr < self.buflen:
            return True
        else:
            self.ptr

File: translation_output/misleading_test/wizardcoder/codenet/python/java-sanitized/p04030.java
import java.util.Scanner;

public class p04030 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        String a = "";
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if (c == 'B') {
                if (!a.isEmpty()) {
                    a = a.substring(0, a.length() - 1);
                }
            } else {
                a += c;
            }
        }
        System.out.println(a);
    }
}